string model_name
float64 new_rate
---
bool success